definitions/ext-istio_service/summary_metrics.yml

```diff
@@ -0,0 +1,31 @@
+clusterName:
+  tag:
+    key: clusterName
+  title: Cluster
+  unit: STRING
+namespaceName:
+  tag:
+    key: namespaceName
+  title: Namespace
+  unit: STRING
+throughput:
+  title: Throughput
+  unit: REQUESTS_PER_SECOND
+  query:
+    select: rate(sum(istio_requests_total), 1 SECOND) WHERE reporter = 'destination'
+    from: Metric
+    eventId: entity.guid
+errorRate:
+  title: Error rate
+  unit: PERCENTAGE
+  query:
+    select: (filter(rate(sum(istio_requests_total), 1 minute), WHERE response_code LIKE '5%') / rate(sum(istio_requests_total), 1 minute)) * 100 WHERE reporter = 'destination'
+    from: Metric
+    eventId: entity.guid
+responseTimeMs:
+  title: Response time (ms)
```

Comment:
should this be `Response time` (without the `ms` designation)?

Reply:
Yep.  Good catch.  Had copied over from the golden metrics (where I followed the convention in [ext-service](https://github.com/newrelic/entity-definitions/blob/a21373c72a60b465c8ec01ad226a90506201a21d/definitions/ext-service/golden_metrics.yml#L33) ).
Will fix